R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -14,8 +14,10 @@
 #' (default \code{rankFromPrefix = FALSE})
 #' 
 #' @param cleanTaxaPattern \code{character} or \code{regex}: If file have
```

Comment:
for function arguments we use style `clean.taxa.pattern`, however here I suggest shorter name like `clean.names`

Reply:
I know we are currently not systematic on this but let us do this at least when we introduce new argument names

Reply:
Sure, I would prefer `clean.taxa.pattern` if it's ok? Because the operation targets only `rowData` not `colData`. Which I have been giving a thought too if necessary to parse it in such a way, sanity check e.g. cleaning numeric column from possible artifacts too and cast it to numeric, perhaps same for  character columns that are e.g supposed to be factors and such. In this case I would see `clean.names` perhaps generic  for  `rowData` and `colData` ofc for example adding an argument which TreeSE (or SE) slot is in question.

Reply:
How about `clean.taxa.names` - could be a bit more intuitive?

The warning message could also mention what value to set for this in case user wants to keep the original names straight from the file?

Reply:
Yes, it is, thanks :) .
Sorry, I might have not understood clearly. But do you mean instead having removed (or substituting with `""`) the patterns mentioned at `clean.taxa.names`, rather add another argument e.g. `taxa.names.replacement` and make the default `taxa.names.replacement=""`?

Reply:
I am not sure if I understand the question but the idea was that if the names would be manipulated in any possible way, then there should be a warning.

Reply:
Ahh ok, yes as I am updating the function now, I am keeping the warnings only when cleaning happens otherwise no warning is thrown.